src/components/Treenav/TreenavCategory.tsx

```diff
@@ -81,7 +83,21 @@ export const TreenavCategory = ({ guide, index, open }) => {
                           isDragDisabled={isDragDisabled}>
                           {provided => {
                             return (
-                              <div css={styles.itemInner} onClick={() => {}}>
+                              <div
+                                css={styles.itemInner}
+                                onClick={() => {
+                                  router.push({
```

Comment:
lets make an handler for this and pass it in something like

`onSectionClick()`

Reply:
assume what i did is what u meant, makes sense in case different types of sections are created